_posts/2023-10-09-Announcing-Data-Prepper-2.5.0.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.5.0"
+authors:
+- graytaylor0
+- oeyh
+date: 2023-10-09 11:30:00 -0500
+categories:
+  - releases
+meta_keywords: Data Prepper, OpenSearch migration, Processors, OpenSearch ingestion
+meta_description: OpenSearch has launched Data Prepper 2.5.0 with feature support for an OpenSearch source, translate and dissect processors, and key-value processor enhancements
```

Comment:
Updated meta with slight tweaks: 

meta_keywords: Data Prepper, dissecting data in OpenSearch, translate and dissect processors OpenSearch, OpenSearch migration

meta_description:  OpenSearch has launched Data Prepper 2.5.0 with feature support for a new OpenSearch source, translate and dissect processors as well as key-value processor enhancements.